### PR TITLE
Fix latest apis row issue on frontpage.

### PR DESCRIPTION
**Pull request by jykae**
I just could not look the overflow of the boxes..

### CODE REVIEW

**Comment by brylie** [comment]
@elnzv or @frenchbread will you please review?

**Comment by elnzv** [comment]
Still have this issue (Chrome & Firefox)

screencapture-localhost-3000-1454062490709

**Comment by brylie** [comment]
Try fixing this strictly with CSS, rather than writing JavaScript. What is the underlying issue? Overflow can be fixed with CSS - and might simply be fixed by allowing the text to wrap.

File: client/views/latest_api/latestApi.js [hunk 1]
```diff
@@ -38,6 +38,15 @@ Template.latestApiBackends.helpers({
       apiBackend.relative_created_at = moment(apiBackend.created_at).fromNow();
     });
 
-    return latestApiBackendsList;
+    // Chunk latestApiBackendsList per rowSize
```
Comment (brylie) [inline client/views/latest_api/latestApi.js]: How can we rely on Bootstrap to wrap the rows? This seems to be a default pattern of CSS/Bootstrap - wrap divs when they exceed the parent container width.

File: client/views/latest_api/latestApi.js [hunk 2]
```diff
@@ -38,6 +38,15 @@ Template.latestApiBackends.helpers({
...
+    chunks = [];
+    rowSize = 4;
```
Comment (brylie) [inline client/views/latest_api/latestApi.js]: This line hard-codes the row size to 4, which is something we have neither designed nor discussed. The template code renders each API Backend panel width at a variable number of rows, depending on screen resolution:

``html
<div class="col-md-3 col-sm-6 hero-feature">
``

Hard-coding this row size is unnecessary, and possibly interferes with our CSS framework.

**Comment by bajiat** [comment]
@jykae Could you also open an issue for this to describe the observed problem you want to solve with this PR?

**Comment by jykae** [comment]
This adds dynamically new row <div class="row">
I noticed behaviour in APIKA when removing and adding API backends. All elements are put in the same row, and the browser don't know how to render when there's no space. Behaviour is unexpected, could go well or be a mess like it was in APIKA frontpage.
Might be related to APIKA styles only because this was not happening in apinf.io.
This fix is in APIKA already, because I wanted it look good in Helsinki.

**Comment by brylie** [comment]
@jykae please open an issue with the above description and screenshot.

**Comment by brylie** [comment]
I think it may be possible to fix this issue by using CSS/HTML. E.g. http://www.bootply.com/jWoHaY7efR

I am not able to find many examples of other people experiencing this sort of layout issue with Bootstrap panels, which leads me to believe there may be some conflict we have introduced in our own code. We should double check whether any of our custom styles are overriding Bootstrap default behavior.

**Comment by jykae** [comment]
Traced the issue with @brylie 
Added issue https://github.com/apinf/api-umbrella-dashboard/issues/824

Trying to find better fix later on with CSS.

**Pull request closed without merging**